src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -115,33 +118,33 @@ public int getTotalMessageCount() {
     PreparedQuery results = datastore.prepare(new Query("Message"));
     return results.countEntities(FetchOptions.Builder.withLimit(1000));
   }
-  
+
   /** Stores the User in Datastore. */
   public void storeUser(User user) {
    Entity userEntity = new Entity("User", user.getEmail());
    userEntity.setProperty("email", user.getEmail());
    userEntity.setProperty("aboutMe", user.getAboutMe());
    datastore.put(userEntity);
   }
-  
+
   /**
    * Returns the User owned by the email address, or
    * null if no matching User was found.
    */
   public User getUser(String email) {
-  
-   Query query = new Query("User")
-     .setFilter(new Query.FilterPredicate("email", FilterOperator.EQUAL, email));
-   PreparedQuery results = datastore.prepare(query);
-   Entity userEntity = results.asSingleEntity();
-   if(userEntity == null) {
-    return null;
-   }
-   
-   String aboutMe = (String) userEntity.getProperty("aboutMe");
-   User user = new User(email, aboutMe);
-   
-   return user;
+
+     Query query = new Query("User")
```

Comment:
Giving variables here more descriptive names than query & results will really improve the future readability of this method, maybe try names like userEmailQuery?

Reply:
changed to userEmailQuery